casper/protocols/blockchain/blockchain_plot_tool.py

```diff
@@ -73,29 +74,26 @@ def get_validator_chains(self):
 
         return vals_chain_edges
 
-    def _track_genesis_linked_messages(self, sent_messages):
-        """Genesis linked messages won't be tracked by communications
-        and need to be manually checked and added"""
-        for sender in sent_messages:
-            message = sent_messages[sender]
-            if message.estimate == self.genesis_block:
-                edge = [self.genesis_block, message]
-                if edge not in self.communications:
-                    self.communications.append(edge)
-
-                self.message_labels[message] = message.sequence_number
+    def _update_new_justifications(self, new_messages):
```

Comment:
We no longer have a notion of direct communications so instead I am showing info on justifications. We only show the dotted line if we have not previously seen a given message in the new messages justification. Keeps the viewgraph less cluttered while still showing all the interesting info